_includes/android/objects.md

```diff
@@ -39,7 +39,7 @@ There are also a few fields you don't need to specify that are provided as a con
 
 ## Retrieving Objects
 
-Saving data to the cloud is fun, but it's even more fun to get that data out again. If you have the `objectId`, you can retrieve the whole `ParseObject` using a `ParseQuery`:
+Saving data to the cloud is fun, but it's even more fun to get that data out again. If you have the `objectId`, which is available once the `ParseObject` been uploaded to the server, you can retrieve the whole `ParseObject` using a `ParseQuery`:
```

Comment:
Directly above this it describes what the objectId is and has an example of saving an object. However, additional clarity doesn't hurt in this instance.

Do you mind changing this for the other platforms? They should have similar wording.

Reply:
Yup. The docs are made noob-friendly, yet I stumbled on this as it sounded kinda like you just need to use `getObjectId()`. I'll update it for the rest of the platforms too.